client/src/components/carousel.jsx

```diff
@@ -1,12 +1,60 @@
+import Thumbnails from './thumbnails.jsx';
+import Scaled from './scaled.jsx';
+import axios from 'axios';
+import styles from './styles.css.js';
+
+
+const dummy = [ //local drive loading for development only
```

Comment:
this is hard coded data from the local drive right?

Reply:
Yep, dont want to use all my S3 bandwidth with constant refreshes